test/integration/api/save.bats

```diff
@@ -42,3 +42,31 @@ function teardown() {
 	rm -f $temp_file_name
 	rm -f $temp_file_name_o
 }
+
+@test "docker save muti-images" {
+	start_docker_with_busybox 1
+	start_docker 1
+	swarm_manage
+	
+	# tag busybox
+	run docker_swarm tag busybox testimage
```

Comment:
So, in order to make sure that `save` actually looks up at different machines I would probably do something like:

```
start_docker_with_busybox 2
docker -H ${HOSTS[0]} tag busybox test1
docker -H ${HOSTS[0]} tag busybox test2
docker_swarm save test1 test2
```

This makes sure `save` is looking at both hosts.